server/services/RollupService.ts

```diff
@@ -280,11 +280,18 @@ export default class RollupService {
         sortField: string;
       };
 
+      const rollupSortMap: { [key: string]: string } = {
+        _id: "rollup.rollup_id.keyword",
+        "rollup.source_index": "rollup.source_index.keyword",
+        "rollup.target_index": "rollup.target_index.keyword",
+        "rollup.rollup.enabled": "rollup.enabled",
```

Comment:
is `rollup.rollup.enabled` being used anywhere? don't see any occurrences in the diff

Reply:
This is used in `Rollups.tsx` when user selects to sort by "enabled". This `const rollupSortMap` is a way that we can quickly parse the chosen field to sort by, and send it as a parameter when calling `getRollups` API.

Reply:
Link to usage of `rollup.rollup.enabled`: https://github.com/opendistro-for-elasticsearch/index-management-kibana-plugin/blob/dab879565a54e83a43f03480f6143c0c3c1ab08b/public/pages/Rollups/containers/Rollups/Rollups.tsx#L363